packages/create-bison-app/template/graphql/modules/user.ts.ejs

```diff
@@ -257,8 +257,9 @@ export const UserCreateInput = inputObjectType({
     t.field('roles', {
       type: list('Role'),
     });
+
     t.field('profile', {
-      type: 'ProfileRelationalCreateInput'
-    })
+      type: 'ProfileRelationalCreateInput',
+    });
```

Comment:
These are changes that ended up in the app after the initial commit (potentially from something like an `eslint --fix`). So just made them here so there aren't uncommitted changes just after creating the app.